dependency_manager/src/edm_tool/__init__.py

```diff
@@ -4,7 +4,8 @@
 #
 """Everest Dependency Manager."""
 from edm_tool import edm
-__version__ = "0.6.2"
+
+__version__ = "0.7.0"
```

Comment:
TODO @hikinggrass probably can just be 0.6.3

Reply:
I used 0.7.0 because there already is a git tag with that value.
It could be confusing to have a 0.6.3 tag newer than 0.7.0

Reply:
Ah I see, then it should probably be something like 0.7.1, I think the 0.7.0 one was created before we decided to use prefixed versions in this repo for everything other than edm.

There still seems to be another issue very similar to the one fixed in this PR, I'll add a commit addressing this to this PR once I have a fix